src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -83,16 +83,12 @@ export default {
 
 		// Add to Basket Interstitial
 		this.initializeAddToBasketInterstitial();
+
 		// Initialize Help Me Choose Experiment
-		if (this.targetedLoanChannel === 'women'
-				|| this.targetedLoanChannel === 'kiva-u-s'
-				|| this.targetedLoanChannel === 'mission-driven-orgs'
-				|| this.targetedLoanChannel === 'short-term-loans'
-		) {
-			this.initializeHelpmeChoose();
-		} else {
-			this.initializeLoanTags();
-		}
+		this.initializeHelpmeChoose();
+
+		// Initialize Loan Tags Experiment
+		this.initializeLoanTags();
```

Comment:
Did we get approval to roll out tags to all categories as well? That's a new result of this.

Reply:
Yeah, Lauren approved this as mentioned in the description

Reply:
Oh oops, I thought that was referring to the help me choose!